playbook-website/app/views/layouts/application.html.erb

```diff
@@ -26,7 +29,7 @@
       <% end %>
     <%end%>
   </body>
-  <%= javascript_packs_with_chunks_tag(*@application_js) %>
```

Comment:
Having two separate calls to the helper makes no sense. Keep it all within `*@application_js`